src/main/java/com/minecolonies/util/BlockUtils.java

```diff
@@ -243,7 +243,7 @@ else if (blockState.getBlock() instanceof BlockSkull)
         }
         else if (blockState.getBlock() instanceof BlockStem)
         {
-            return ((BlockStem) blockState.getBlock()).getSeedItem();
+            //return ((BlockStem) blockState.getBlock()).getSeedItem();
```

Comment:
![MAJOR](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-major.png) This block of commented-out lines of code should be removed. [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](http://home.kk-sc.de:9000/coding_rules#rule_key=squid%3ACommentedOutCodeLine)